tests/cli.ts

```diff
@@ -114,6 +114,94 @@ const skip = Number(major) < 20
         ['[{"hello":"world"},42]']
       )
     })
+    describe('--indent', () => {
+      ok(
+        'basic',
+        'hello:\n  world: 2',
+        ['--indent', '3'],
+        ['hello:\n   world: 2']
+      )
+      ok(
+        '--json',
+        'hello: world',
+        ['--json', '--indent', '2'],
+        ['[\n  {\n    "hello": "world"\n  }\n]']
+      )
+      ok(
+        '--single',
+        'hello: world',
+        ['--json', '--indent', '2', '--single'],
+        ['{\n  "hello": "world"\n}']
+      )
+      ok(
+        'multiple',
+        'hello: world\n---\n42',
+        ['--json', '--indent', '2'],
+        ['[\n  {\n    "hello": "world"\n  },\n  42\n]']
+      )
+      ok(
+        'Lexer',
+        'hello: world',
+        ['lex', '--json', '--indent', '2'],
+        ['[\n  "\\u0002",\n  "\\u001f",\n  "hello",\n  ":",\n  " ",\n  "\\u001f",\n  "world"\n]']
+      )
+      ok(
+        'CST parser',
+        'hello: world',
```

Comment:
The expected output includes a trailing newline, which is missing from here.

```suggestion
        'hello: world\n',
```